.ci/validation.sh

```diff
@@ -650,6 +650,22 @@ javac22)
   fi
   ;;
 
+javac25)
+  files=($(grep -Rli --include='*.java' ': Compilable with Java25' \
```

Comment:
Only change is that I added `-i`(`--ignore-case`). See issue below as to why I am doing it:
- https://github.com/checkstyle/checkstyle/issues/18092